exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/StandardServiceConfigurationTestKitTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.testkit;
+
+import static com.exonum.binding.testkit.TestKitTestUtils.createArtifact;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.exonum.binding.common.hash.HashCode;
+import com.exonum.binding.common.hash.Hashing;
+import com.exonum.binding.common.messages.Service.ServiceConfiguration;
+import com.exonum.binding.common.messages.Service.ServiceConfiguration.Format;
+import com.exonum.binding.core.runtime.ServiceArtifactId;
+import com.exonum.binding.core.storage.database.Snapshot;
+import com.exonum.binding.core.storage.indices.ProofMapIndexProxy;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+public class StandardServiceConfigurationTestKitTest {
```

Comment:
Why is this test added to testkit — it checks some runtime-specific functionality,
that does not affect either testkit or the core? May be in integration-tests. 

However, I'd reconsider if it is needed — as this functionality is runtime-specific and based on the same mechanism that is already tested (= passing config parameters from test thru testkit, core, to runtime); but testing it at this level is rather complex.

Reply:
> However, I'd reconsider if it is needed — as this functionality is runtime-specific and based on the same mechanism that is already tested (= passing config parameters from test thru testkit, core, to runtime); but testing it at this level is rather complex.

I am not convinced its value justifies the cost.